### ISSUE TITLE
Design for Feedback options on the project index page

### ISSUE THREAD

**Issue by gabsie** [ebi-ait/dcp-ingest-central#127]
User story:
As a single cell researcher, I would like to Get in touch with this resource, so that I can: 
	A. Submit my project to the list 
	C. I want to ask a question about the criteria, process, etc
	D. I want to provide feedback.
	E. I want to correct something in the project I see.

**Comment by mshadbolt** [comment]
I quite like having a simple thing that pops out from the side rather than a whole new page, e.g. on the gwas catalog https://www.ebi.ac.uk/gwas/

could be a good first easy option since we will probably just be wanting people to email wrangler email address in the first instance

**Comment by mshadbolt** [comment]
In meeting today we discussed having something similar (but maybe better looking) that sits on the side of the browser and follows the user as they scroll. Initially this can simply be an email to wrangler-team@data.humancellatlas.org

@gabsie to give input on what the button might look like

Then @jacobwindsor to implement

Eventually we may want to build a custom form like europmc http://europepmc.org/feedback?url=https://europepmc.org/
A google or other form
Or follow what covid portal did using the EBI services support page https://www.ebi.ac.uk/support/covid19dataportal

**Comment by lauraclarke** [comment]
Is there a reason not to connect it to project registration rather than build a separate solution?

**Comment by mshadbolt** [comment]
I think for now we are thinking about it in the simplest way possible that would capture feedback and if someone wants their project to appear, i.e. a button that is a `mailto:` link

Totally agree that if we implement the 'I want to register a project' thing it should just link to registering a project in ingest, but I guess that is blocked right now until we are allowed to give the general public the option to see the ui.

For the alternative forms I listed I was thinking more if people want to send general feedback/queries or errors rather than adding a project

**Comment by gabsie** [comment]
Hi, @mshadbolt @jacobwindsor 

I have attached the updated layout. I have included a button on the side for feedback, and a line at the bottom of the table. For now if we're about to launch, let's have those send emails.

I will also attach this design to the other layout ticket.![Current variant - Project Catalogue.png](https://images.zenhubusercontent.com/5b18f77c348b7966b4345c29/d4a3ef4a-d487-427e-8acd-862cdc9564da)

(I am also investigating with webdev team about what forms we have implemented for others)

Gabs
